package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "muya",
+    "name": "@langh/muya",
```

Comment:
Why change the name?

Reply:
Sorry, I made a wrong push after submitting the pull request.